.travis.yml

```diff
@@ -14,16 +14,12 @@ cache:
 
 matrix:
   include:
-    # Symfony 2.7.0
-#    - php: 5.5
-#      env: COMPOSER_FLAGS="--prefer-lowest"
-    # Symfony 2.8
-#    - php: 5.6
-#      env: SYMFONY_VERSION="2.8.*@dev"
-    # Symfony 3.0
-    - php: 5.6
+    - php: 7.0
       env: DEPS=dev
 
+env:
+  - COMPOSER_FLAGS="--prefer-lowest"
```

Comment:
please do this in the matrix. see https://github.com/yannickl88/form-handler-bundle/blob/master/.travis.yml

you are starting only one build now. what I would like:
```yaml
env:
  - COMPOSER_FLAGS="--prefer-stable"

php:
    - '7.0'
    - '7.1'
    - nightly

matrix:
  include:
    - php: 7.0
      env: COMPOSER_FLAGS="--prefer-lowest"
    - php: 7.0
      env: COMPOSER_FLAGS=""
    - php: 7.1
      env: COMPOSER_FLAGS="--prefer-lowest"
    - php: 7.1
      env: COMPOSER_FLAGS=""
  allow_failures:
    - php: nightly
```